lib/puppet.rb

```diff
@@ -20,8 +20,20 @@
 require 'puppet/external/pson/common'
 require 'puppet/external/pson/version'
 require 'puppet/external/pson/pure'
-require 'gettext-setup'
-require 'locale'
+
+# When running within pupeptserver, the gettext-setup gem is not available, so
```

Comment:
also I this comment might be more accurate as "the gettext-setup gem might not be available"